alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsVerticalScaling.md

```diff
@@ -0,0 +1,44 @@
+# MDSCpuUsageHighNeedsVerticalScaling
+
+## Meaning
+
+Ceph MDS cpu usage for the MDS daemon has exceeded the threshold for adequate
```

Comment:
Type: "cpu" is used in capital letters in all other places

Reply:
Fixed... Thanks